Arrowgene.Ddon.Database/DdonDatabaseBuilder.cs

```diff
@@ -13,7 +13,7 @@ public static class DdonDatabaseBuilder
 {
     private const string DefaultSchemaFile = "Script/schema_sqlite.sql";
 
-    public const uint Version = 42;
+    public const uint Version = 43;
```

Comment:
There's another PR using this version number, we have to keep in mind before merging